bundles/org.openhab.binding.volumio/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -31,23 +31,24 @@
 				<label>Hostname</label>
 				<description>The hostname of your Volumio device</description>
 			</parameter>
-			<parameter name="port" type="integer" required="true">
+			<parameter name="port" type="integer" required="false">
 				<label>Port</label>
 				<description>The port of your Volumio device (default is 3000)</description>
 				<default>3000</default>
 			</parameter>
-			<parameter name="protocol" type="text" required="true">
+			<parameter name="protocol" type="text" required="false">
```

Comment:
I have seen several bindings handle this different. If a default value is added does that mean that required should also be set to false?
I always considered the parameter to be requried independent on if the parameter has an option list and/or default value. 
My assumprtion is that when requried = false and there is an option list and default, a user is still able to deselect the default option in the UI and save the configuration.

Reply:
I believe otherwise you cannot omit the parameter in a .things file, please correct me if I'm wrong @lolodomo.

Reply:
You made me in doubt ! Your use case with UI should be tested again

I suggested my approach many many times when I reviewed PRs during the last years, so if I was wrong, that is a disaster!

> I believe otherwise you cannot omit the parameter in a .things file, please correct me if I'm wrong @lolodomo.

Maybe. But I am not 100% sure now.

Reply:
Sorry if I cause confusion.

Reply:
Ok, did some testing with MainUI:

Case : required is true + default is set
Result: Config field is set to default immediate after removing the value and leaving input field. The input inner text is red colored. Also adittional red text "Field is required" is shown.

Case : required is false +  default is set
Result: Config field is set to default immediate after removing the value and leaving input field. No additional UI hints (color or text) are shown. 

Textual config is not affected. 

So my conclusion is that there is little effect on the current UI's. So nothing to really worry about. But i would also recomment to set required=true when the binding needs it (wether there is a default or not). The current UI already handles this a little different and future UI's could make use of this flag. In essence the value remains required.